### PR TITLE
Adding py_tsne package

### PR DESCRIPTION
**Pull request by WiscEvan**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by WiscEvan** [comment]
I'm willing to be listed as a maintainer

**Comment by WiscEvan** [comment]
@danielfrg Can you comment that you are willing to be listed as a maintainer? Thanks!

**Comment by danielfrg** [comment]
I'm willing to be listed as a maintainer.

**Comment by WiscEvan** [comment]
@conda-forge-admin, please ping team

File: recipes/tsne/setup.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
+"""
```
Comment (jakirkham) [inline recipes/tsne/setup.py:1]: Is this file actually needed?

File: recipes/tsne/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/WiscEvan/tsne/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py<37]
+  skip: false  # [py==27]
```
Comment (marcelotrevisani) [inline recipes/tsne/meta.yaml:16]: Please remove this line, it is not necessary
```suggestion
```

**Comment by danielfrg** [comment]
Ah, I think we are adding the setup.py because we have not released it on pypi with support for python 3.8 right @WiscEvan? If thats the reason I can release something maybe today.

File: recipes/tsne/meta.yaml (hunk 2 above)
Comment (WiscEvan) [reply recipes/tsne/meta.yaml:16]: I thought `# [py<37]` would include 3.7 as well as 3.8. Is this incorrect?

@danielfrg, an updated release would be useful. I have removed `extra_link_args = ['-lsatlas']`  from the linux distribution checks within `setup.py`, the tests seem to pass with warnings using pytest. I can submit these changes to you as a PR from my repo if necessary.

**Comment by WiscEvan** [comment]
@danielfrg, an updated release would be useful. I have removed `extra_link_args = ['-lsatlas']`  from the linux distribution checks within `setup.py`, the tests seem to pass with warnings using pytest. I can submit these changes to you as a PR from my repo if necessary.

**Comment by danielfrg** [comment]
Ok, yeah if you submit that as a PR I can build a release after!

File: recipes/tsne/meta.yaml (hunk 2 above)
Comment (marcelotrevisani) [inline recipes/tsne/meta.yaml:16]: I think that should be then:
```suggestion
  skip: true  # [py3k and py<37]
```

File: recipes/tsne/setup.py (hunk 1 above)
Comment (WiscEvan) [reply recipes/tsne/setup.py:1]: I've removed the file and will update when @danielfrg has released the next version.

**Comment by danielfrg** [comment]
I released version `0.2.0` with Python 3 support, bumping versions here will probably make it work.

**Comment by WiscEvan** [comment]
@conda-forge-admin, please ping team

**Comment by WiscEvan** [comment]
@conda-forge/staged-recipes can this PR be merged or are there additional comments that need to be addressed?

File: recipes/tsne/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - setuptools
+    - cython
+    - numpy
+    - openblas
```
Comment (isuruf) [inline recipes/tsne/meta.yaml:28]: See https://conda-forge.org/docs/maintainer/knowledge_base.html#blas

**Comment by WiscEvan** [comment]
@conda-forge-admin, please ping team

File: recipes/tsne/meta.yaml (hunk 3 above)
Comment (WiscEvan) [reply recipes/tsne/meta.yaml]: I think I have addressed the blas issue. Is this now ready to be merged?

File: recipes/tsne/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/4c/ea/f4deb02eb49bbc7006624398d3909bbb43cd93efd58e66b74320b5530edd/tsne-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py2k]
+  skip: true  # [win]
```
Comment (scopatz) [inline recipes/tsne/meta.yaml:16]: Please combine these line to `[py2k or win]`

File: recipes/tsne/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - cython
+    - libcblas
+    - python
+    - pip
+    - setuptools
```
Comment (scopatz) [inline recipes/tsne/meta.yaml:28]: You should only need pip, which implies setuptools

**Comment by WiscEvan** [comment]
@conda-forge-admin, please ping team
